mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterConfigAdd.java

```diff
@@ -35,6 +35,7 @@
     static final ModClusterConfigAdd INSTANCE = new ModClusterConfigAdd();
 
     private ModClusterConfigAdd() {
+        super(ModClusterConfigResourceDefinition.ATTRIBUTES);
```

Comment:
If you do this (which you should; it's best practice) then the populateModel impl below should be changed. It loops through the attributes in ModClusterConfigResourceDefinition.ATTRIBUTES. That can be replaced with a call to super.populateModel(....).

I'm not sure why this class treats ModClusterConfigResourceDefinition.SIMPLE_LOAD_PROVIDER differently.